examples/systemd/rest-server.service

```diff
@@ -15,10 +14,8 @@ ExecStart=/usr/local/bin/rest-server --path /path/to/backups
 Restart=always
 RestartSec=5
 
-# The following options are available (in systemd v247) to restrict the
-# actions of the rest-server.
-
-# As a whole, the purpose of these are to provide an additional layer of
+# The following options restricts the actions of the rest-server.
```

Comment:
The systemd version number was added some time ago as some of the used options are not available in older systemd versions. Thus, I'd prefer to keep it.

Reply:
Done